int32 channel
int32 minPulseWidth
float64 minRadians
int32 maxPulseWidth
float64 maxRadians
---
int32 errorCode
